src/assets/sass/modules/Button/_button.scss

```diff
@@ -0,0 +1,7 @@
+.btn{
+    border: 0;
+    outline: 0;
+    background: $shamrock;
+    color: $white;
+    padding: 5px 8px;
+}
```

Comment:
Add a new line after the end of the codebase

Reply:
🆗